trace/cloud-trace-demo-app/README.md

```diff
@@ -0,0 +1,44 @@
+# cloud-trace-demo-app
+
+Open this demo app in [Google Cloud Shell](https://cloud.google.com/shell/docs/). This includes necessary tools.
+
+We provide a public image for the services in this demo app. You could also build
+your own following step 4 - 6.
+
+[![Open Cloud Trace Demo APP in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=trace/cloud-trace-demo-app/README.md&amp;cloudshell_tutorial=README.md)
+
+#### Demo Requirements
+If you are using Cloud Shell, skip to the next section.
+
+1. Install gcloud <https://cloud.google.com/sdk/install>
+2. Install kubectl <https://kubernetes.io/docs/tasks/tools/install-kubectl/>
+3. Install docker <https://docs.docker.com/install/>
+
+#### Google Container Registry Image Setup
+If you are using the provided image, skip to the next section.
+
+4. Get default project id and set environment variable:
+    `PROJECT_ID=$(gcloud config get-value project)`
+5. Build Image:
+    `docker build -t gcr.io/${PROJECT-ID}/cloud-trace-demo .`
+6. Upload Image to Container Registry:
+    `gcloud docker -- push gcr.io/${PROJECT-ID}/cloud-trace-demo-test:v1`
+
+#### Create a GKE cluster
+7. Enable Google Cloud and set up region and zone.
+    `gcloud init`
+8. Enable the GKE api & billing:
+    `gcloud services enable container.googleapis.com`
+9. Create a GKE cluster named "demo":
+    `gcloud container clusters create demo`
+
+#### Send Requests to See Generated Traces
+
+10. Run setup.sh to apply the YAML files. Note this file configures the default image we provide.
+    If you are using your own image, please change the variable in the [YAML](./app/demo-service-a.yaml) file and in the templates[B](./app/demo-service-b.yaml.template)[C](./app/demo-service-c.yaml.template).
```

Comment:
hyperlink nit -
10. Run setup.sh to apply the YAML files. Note this file configures the default image we provide.
 If you are using your own image, change the `image` variable in the following files:
-  [YAML  file](./app/demo-service-a.yaml)
-  [template B](./app/demo-service-b.yaml.template) 
-  [template C](./app/demo-service-c.yaml.template)

Reply:
I would actually consider making the "if you are using your own image part" step 10, and then telling users they can skip to the next step if they're using the default image, with the new next step being to run setup.sh

Reply:
Done. Thanks for the suggestion.